src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/SignatureContext.cs

```diff
@@ -83,5 +83,22 @@ public ModuleToken GetModuleTokenForField(FieldDesc field, bool throwIfNotFound
         {
             return Resolver.GetModuleTokenForField(field, throwIfNotFound);
         }
+
+        public bool Equals(SignatureContext other)
```

Comment:
Nit: there shouldn't be a need for this, and we can just have the Equals(object) API. The other key structs have it because they implement IEquatable<T>, because they are all structs and the IEquatable<T> is the way to avoid boxing/unboxing, but SignatureContext is a class.
It's fine to leave this the way it is. This is just a FYI comment